samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemAddListener.java

```diff
@@ -866,17 +859,17 @@ else if(item.getTypeId().equals(TypeFacade.IMAGEMAP_QUESTION)) {
             item.setItemMetaDataSet(prepareMetaData(item, bean));
 
             // prepare feedback, only store if feedbacks are not empty
-            if ( (bean.getCorrFeedback() != null) &&
-      			  (!bean.getCorrFeedback().equals(""))) {
-            	item.setCorrectItemFeedback(stripPtags(bean.getCorrFeedback()));
+      	  	if (!StringUtils.isEmpty(bean.getCorrFeedback())) {
```

Comment:
Better using 'isNotEmpty' for more readability

Reply:
Done